lib/create.js

```diff
@@ -200,10 +200,12 @@ module.exports.deleteInvalidTemplate = function(options) {
  *     - `e` {Error} is null unless there was an error.
  */
 
-module.exports.downloadTemplate = function(options, callback) {
-    var destinationPath = module.exports.templatePath(options),
-        url = 'https://github.com/phonegap/phonegap-app-hello-world/archive/master.tar.gz';
-
+module.exports.downloadTemplate = function(options, callback, url) {
+    var destinationPath = module.exports.templatePath(options);
+	if (url === undefined) {
+        var url = 'https://github.com/phonegap/phonegap-app-hello-world/archive/master.tar.gz';
+    }
+    
```

Comment:
That seems fine. I know the PhoneGap Desktop App is going to support multiple templates in the near future and was addressing that issue early.